kore/src/Debug.hs

```diff
@@ -316,8 +314,18 @@ datatypeInfoCofreeF =
         "Control.Comonad.Trans.Cofree"
         "CofreeF"
         (constrInfo :* Nil)
+        (SOP.POP
+            ((strictnessInfoForX :* (strictnessInfoForY :* Nil)) :* Nil)
+        )
   where
     constrInfo = SOP.Infix ":<" SOP.RightAssociative 5
+    -- TODO: is the strictness info correct?
+    strictnessInfoForX =
+        SOPMeta.StrictnessInfo
+            SOPMeta.NoSourceUnpackedness
+            SOPMeta.NoSourceStrictness
+            SOPMeta.DecidedLazy
```

Comment:
The strictness info is arbitrary because it doesn't affect how we print anything. It would be better to change `debugPrecAux` to take a custom type with only the required information.